packages/@angular/cli/commands/completion.ts

```diff
@@ -7,7 +7,7 @@ const stringUtils = require('ember-cli-string-utils');
 const Command = require('../ember-cli/lib/models/command');
 const lookupCommand = require('../ember-cli/lib/cli/lookup-command');
 
-function extractOptions(opts: any): string {
+function extractOptions(opts: any): string[] {
   const output: string[] = [];
 
   for (let index = 0; index < opts.length; index++) {
```

Comment:
The loop inside extractOptions adds the "--" prefixes and the aliases. What do you mean?